framework/src/modules/chain/logic/init_transaction.js

```diff
@@ -163,41 +49,21 @@ class Transaction {
 
 	// TODO: remove after https://github.com/LiskHQ/lisk/issues/2424
 	dbRead(raw) {
-		if (!raw.t_id) {
+		if (
+			raw.t_id === undefined ||
+			raw.t_id === null ||
+			raw.t_type === undefined ||
+			raw.t_type === null
```

Comment:
falsy check `(!raw.t_id  || !raw.t_type)` is not enough?

Reply:
!raw.t_type would evaluate to true if type = 0

Reply:
ah yes... sorry ='(